pkgs/desktops/gnome-3/3.24/core/gnome-shell/default.nix

```diff
@@ -28,7 +28,7 @@ in stdenv.mkDerivation rec {
       libgweather # not declared at build time, but typelib is needed at runtime
       gnome3.gnome-clocks # schemas needed
       at_spi2_core upower ibus gnome_desktop telepathy_logger gnome3.gnome_settings_daemon
-      pythonEnv gobjectIntrospection (stdenv.lib.getLib dconf) ];
+      pythonEnv gobjectIntrospection dconf ];
```

Comment:
Ah, thanks. Reverted. I think I lost track a bit as to what should change. I went through c9d419a22bc42f79acfdf766b9d5908cf12ccdbc and re-applied everything I could find.